app/exec/extension/install.ts

```diff
@@ -0,0 +1,489 @@
+import { TfCommand } from "../../lib/tfcommand";
```

Comment:
Use tabs to match the rest of the files' indentation method. If you are passionate about using spaces, let's fight about it!